src/apps/search/panels/BasePanel.tsx

```diff
@@ -165,6 +171,17 @@ const BasePanel = (props: Props) => {
   },
   [data, props.name]);
 
+  /**
+   * Memo-izes the cover URL.
+   */
+  const coverUrl = useMemo(() => {
+    if (!mediaContentsLoading && mediaContents.length > 0) {
+      return mediaContents[0].content_url
```

Comment:
See [comment](https://github.com/performant-software/core-data-places/pull/185/files#r2050702733) about `content_url` in #185. Addiontally, to address any loading lag, we could render a gray placeholder.

Reply:
Would it be alright to show the placeholder for records that don't have any images?

If not, the placeholder would flash and then disappear for those records because we can't know whether a record has images until we load from the `media_contents` endpoint.

Reply:
Hm, that's a good question. Could we render the placeholder after we've loaded the records from the `media_contents` endpoint? I suspect the request from the `<img` tag is what is taking a long time.

Reply:
Hm, so we would watch the `complete` attribute of the `<img>` element and display the placeholder until `complete = true`?

Reply:
The main issue above I think is the layout shift where the image element is only inserted once we know the record has associated media. That means the rest of the panel's content jumps up and down as we move between records that have media and those that don't.